admin/api_cloud_provider_access.go

```diff
@@ -23,7 +23,7 @@ type CloudProviderAccessApi interface {
 		@param roleId Unique 24-hexadecimal digit string that identifies the role.
 		@return AuthorizeCloudProviderAccessRoleApiRequest
 	*/
-	AuthorizeCloudProviderAccessRole(ctx context.Context, groupId string, roleId string, cloudProviderAccessRole *CloudProviderAccessRole) AuthorizeCloudProviderAccessRoleApiRequest
+	AuthorizeCloudProviderAccessRole(ctx context.Context, groupId string, roleId string, cloudProviderAccessAWSIAMRoleUpdateRequest *CloudProviderAccessAWSIAMRoleUpdateRequest) AuthorizeCloudProviderAccessRoleApiRequest
```

Comment:
Properties related to azure provider have been removed, https://jira.mongodb.org/browse/CLOUDP-251644 is in progress to fix this.